### PR TITLE
Update website landing page

### PR DESCRIPTION
**Pull request by jshaughn**
Fixes https://github.com/kiali/kiali/issues/2689

@maltron @lucasponce , I tried a few suggestions from the issue comments and for me this is what looked best.  Note that this also tweaks the questions and the "about" blurb.   

As an aside this update the video section to include the most recent sprint demos.

### CODE REVIEW

File: themes/kiali/layouts/partials/home/about-section.html [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
     <p>
-    Kiali is an observability console for Istio with service mesh configuration
-    capabilities. It helps you to understand the structure of your service mesh
-    by inferring the topology, and also provides the health of your mesh. Kiali
-    provides detailed metrics, and a basic Grafana integration is available for
-    advanced queries. Distributed tracing is provided by integrating Jaeger.
+    Kiali is a management console for Istio-based service mesh.  It provides
+    dashboard, observability and configuration capabilities. It helps you to
+    understand the structure of your service mesh by inferring the topology and
+    also displays the health of your mesh. Kiali provides detailed metrics,
```
Comment (lucasponce) [inline themes/kiali/layouts/partials/home/about-section.html:8]: I miss the verb "operate" the Service Mesh.
Some way that is clear that Kiali performs actions that involve complex configuration to allow manage Istio.

File: themes/kiali/layouts/partials/home/hero-section.html [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-    <h1>Service mesh observability<br />and configuration</h1>
+    <h1>Service mesh management<br />for Istio</h1>
 
     <p>
     <strong>Kiali answers the questions:</strong><br />
-    What microservices are part of my Istio service mesh?<br />
+    Which microservices are part of my service mesh?<br />
     How are they connected?<br />
     How are they performing?<br />
+    How can I configure them?<br />
```
Comment (lucasponce) [inline themes/kiali/layouts/partials/home/hero-section.html:9]: I'd like to use a high level word "operate" I think.
But I let you to re-phrase this.

File: themes/kiali/layouts/partials/home/about-section.html (hunk 1 above)
Comment (jshaughn) [reply themes/kiali/layouts/partials/home/about-section.html:8]: I like the term "management console", let me then also work in the  "operate" verb:  How about:

> Kiali is a management console for Istio-based service mesh.  It provides
> dashboards, observability and lets you to operate your mesh with robust
> configuration and validation capabilities.  It shows the structure of your
> service mesh by inferring traffic topology and displays the health of your
> mesh.  Kiali provides detailed metrics, ...

File: themes/kiali/layouts/partials/home/hero-section.html (hunk 2 above)
Comment (jshaughn) [reply themes/kiali/layouts/partials/home/hero-section.html:9]: How about replacing, "How can I configure them"  with "How can I operate on them"?

File: themes/kiali/layouts/partials/home/about-section.html (hunk 1 above)
Comment (lucasponce) [reply themes/kiali/layouts/partials/home/about-section.html:8]: It sounds great to me.

File: themes/kiali/layouts/partials/home/hero-section.html (hunk 2 above)
Comment (lucasponce) [reply themes/kiali/layouts/partials/home/hero-section.html:9]: +1

**Pull request merged**